app/javascript/components/pages/admin/usage-reports/export/component.jsx

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) 2014 - 2023 UNICEF. All rights reserved.
+
+import css from "./styles.css";
+import PropTypes from "prop-types";
+import { useDispatch } from "react-redux";
+import { useState } from "react";
+import ActionDialog from "../../../../../components/action-dialog";
+import Form from "../../../../../components/form";
+
+import { NAME, FORM_ID } from "./constants";
+import { form } from "./form";
+import { saveExport } from "../../../../../components/record-actions/exports/action-creators";
+import { formatFileName } from "../../../../../components/record-actions/exports/utils";
+
+
+const Component = ({ close, i18n, open, pending, setPending }) => {
+  const dispatch = useDispatch();
+  const dialogPending = typeof pending === "object" ? pending.get("pending") : pending;
+  const message = undefined;
+
+  // Grouping useState variables together
+  const [validationError, setValidationError] = useState("");
+
+  // Form submission
+  const onSubmit = getData => {
+    // Check if any of the required fields are null
+    if (!getData.From) {
```

Comment:
Use yup for validation instead of the following. You can pass a schema into like 
```
import { object, string } from "yup";

export const validationSchema = object().shape({
  message: string().required()
});

<Form validations={validationSchema} />
```
into the form component

https://github.com/jquense/yup